js/components/frame.js

```diff
@@ -773,6 +774,10 @@ class Frame extends ImmutableComponent {
         this.webview.focus()
       }
       windowActions.setNavigated(e.url, this.props.frameKey, false)
+      // After navigating to the URL, set correct frame title
+      let webContents = this.webview.getWebContents()
+      let title = webContents.getTitleAtIndex(webContents.getCurrentEntryIndex())
```

Comment:
@bsclifton - if you add these methods to https://github.com/brave/electron/blob/master/lib/renderer/web-view/web-view.js#L391 they can be called directly on the webview